client/.storybook/main.js

```diff
@@ -2,15 +2,33 @@ module.exports = {
   "stories": [
     "../src/**/*.stories.@(js|jsx|ts|tsx)"
   ],
-  "addons": [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials"
-  ],
```

Comment:
I’ll be doing much more work with Storybook in a subsequent PR – just removing those for now as they’re largely unused currently.